mindsdb/integrations/utilities/rag/pipelines/rag.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, retriever_runnable, prompt_template, llm, reranker: bool = DE
         self.prompt_template = prompt_template
         self.llm = llm
         if reranker:
-            self.reranker = OpenAIReranker()
+            self.reranker = LLMReranker()
```

Comment:
we should allow users to use non default args for reranker, so entry point here and perhaps pass in via [RagPipelineModel](https://github.com/mindsdb/mindsdb/blob/0c03525cdd8ab2c1ec17414be64aa5f11b792893/mindsdb/integrations/utilities/rag/settings.py#L104)

Reply:
do I pass arguments as parameters of the class constructor, I should I pass them in some other way?

Reply:
like:
```LLMReranker(model_name="llama7b", base_url="http://localhost/v1")```
or should it be some other way to pass that data?

Reply:
Yes via constructor is good, and in the rag pipeline we pass them in from config.

Reply:
updated, please take a look @dusvyat